README.md

```diff
@@ -25,7 +25,7 @@ This repository holds the kcp tests that tests against the publicly available  i
   ```console
   $ kubectl oidc-login get-token --oidc-issuer-url=<oidc issuer url> --oidc-client-id=<oidc client ID> --oidc-redirect-url-hostname=127.0.0.1
   ```
-  - For kcp services deploy with redhat sso( **Only supported kcp services deploy with redhat sso** ), there's another login method by use `offline token`, you could get an offline token ( [How to generate an offline token](https://access.redhat.com/articles/3626371#bgenerating-a-new-offline-tokenb-3) ) and log in to the kcp service by execute the follow commands.
+  - For kcp services deploy with redhat sso( **Only supported kcp services deploy with redhat sso** ), there's another login method called `login with offline token` that could be used, you could get an offline token ( [How to generate an offline token](https://access.redhat.com/articles/3626371#bgenerating-a-new-offline-tokenb-3) ) and log in to the kcp service by executing the following commands without inputting credentials into the browser.
```

Comment:
@Phaow still see text in the bold here, also s/without inputting/ with out having to input

Reply:
Sorry, I forgot update it, addressed.

Reply:
Thanks, @Phaow sorry if my suggestion was not clear, what i meant was to remove the text in bold completely.

Reply:
np, addressed, I misunderstand.